core/templates/pages/exploration-editor-page/editor-tab/templates/modal-templates/customize-interaction-modal.controller.ts

```diff
@@ -219,9 +240,69 @@ angular.module('oppia').controller('CustomizeInteractionModalController', [
       }
     };
 
+    /**
+     * The default values of SubtitledHtml and SubtitledUnicode objects in the
+     * customization arguments have a null content_id. This function populates
+     * these null content_id's with a content_id generated from traversing the
+     * schema and with next content id index, ensuring a unique content_id.
+     */
+    $scope.populateNullContentIds = function() {
+      const interactionId = $scope.StateInteractionIdService.displayed;
+
+      let traverseSchemaAndAssignContentIds = (
+          value: InteractionCustomizationArgsValue,
+          schema: Schema,
+          currentContentId: string,
+      ): void => {
+        if (schema.type === SchemaConstants.SCHEMA_KEY_LIST) {
+          for (
+            let i = 0;
+            i < (<InteractionCustomizationArgsValue[]> value).length;
+            i++
+          ) {
+            traverseSchemaAndAssignContentIds(
+              value[i],
+              <Schema> schema.items,
+              `${currentContentId}`);
+          }
+        } else if (schema.type === SchemaConstants.SCHEMA_TYPE_DICT) {
+          schema.properties.forEach(property => {
+            const name = property.name;
+            traverseSchemaAndAssignContentIds(
+              value[name],
+              property.schema,
+              `${currentContentId}_${name}`);
+          });
+        } else if (schema.type === SchemaConstants.SCHEMA_TYPE_CUSTOM &&
+            ((schema.obj_type ===
+              SchemaConstants.SCHEMA_OBJ_TYPE_SUBTITLED_UNICODE) ||
+            schema.obj_type === SchemaConstants.SCHEMA_OBJ_TYPE_SUBTITLED_HTML)
+        ) {
+          if ((<SubtitledHtml|SubtitledUnicode>value).getContentId() === null) {
+            (<SubtitledHtml|SubtitledUnicode>value).setContentId(
+              `${currentContentId}_${StateNextContentIdIndexService.displayed}`
+            );
+            StateNextContentIdIndexService.displayed += 1;
+          }
+        }
+      };
+
+      const caSpecs = INTERACTION_SPECS[interactionId].customization_arg_specs;
+      const caValues = StateCustomizationArgsService.displayed;
+      for (const caSpec of caSpecs) {
+        const name = caSpec.name;
+        if (name in caValues) {
```

Comment:
Prefer hasOwnProperty (ditto if you've used this construct elsewhere).

Reply:
Done